addons/hr_holidays/migrations/12.0.1.5/openupgrade_analysis_work.txt

```diff
@@ -0,0 +1,242 @@
+---Fields in module 'hr_holidays'---
+hr_holidays  / hr.employee              / current_leave_id (many2one)   : relation is now 'hr.leave.type' ('hr.holidays.status')
+# NOTHING TO DO: handled in the model rename
+
+*** new hr.leave model ***
+# DONE: post-migration: filled from hr.holidays  (when old type is 'remove')
+
+hr_holidays  / hr.holidays              / date_from (datetime)          : now required, default = function
+hr_holidays  / hr.holidays              / date_to (datetime)            : now required, default = function
+DONE: post-migration: filled if empty
+
+hr_holidays  / hr.holidays              / holiday_status_id (many2one)  : relation is now 'hr.leave.type' ('hr.holidays.status')
+hr_holidays  / hr.holidays              / holiday_type (selection)      : selection_keys is now '['category', 'company', 'department', 'employee']' ('['category', 'employee']')
+hr_holidays  / hr.holidays              / linked_request_ids (one2many) : relation is now 'hr.leave' ('hr.holidays')
```

Comment:
Should this be reported if we have the renaming clear (that's why the model name appears `hr.holidays` and not `hr.leave`)?

Reply:
What do you mean? The `one2many` always are `#NothingToDo`

Reply:
I'm talking about `openupgrade_records` module.

Reply:
Ah. Despite the model renaming, the `openupgrade_records` always report the model change in the x2x fields.

Reply:
Yes, and this is what I'm proposing to improve for cleaning this file with not needed things to check.

Reply:
Ok, but I prefer having that in the report

Reply:
Why? This is going to be always "Nothing to do". Remember that the goal of this file is to highlight things that may have something to do.